### ISSUE TITLE
Long project / dataset filenames cause visual problems

### ISSUE THREAD

**Issue by alex-r-bigelow** [Kitware/candela#254]
Filenames can overlap each other in both the dataset / project selection menus, as well as the toggle buttons on top of the widgets

**Comment by ronichoudhury** [comment]
Can we solve both problems with judicious use of ellipses? Maybe with either a tooltip or some kind of temporary animation revealing the full name on mouseover or hover?

**Comment by alex-r-bigelow** [comment]
Yup, that's the plan. A convenient time to fix this will be when we get around to tackling #164

**Comment by TristanWright** [comment]
You could use `text-overflow: ellipsis;` in combination with some tooltip or just a `title` attribute.

https://codepen.io/tristaaan/pen/VKvvoX?editors=1100
